### PR TITLE
PXC-4485 PXC 5.7.44-31.65-4 release notes and release note tasks

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch pxc-4485
	new file:   docs/release-notes/5.7.44-31.65.4.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/5.7.44-31.65.4.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Percona XtraDB Cluster 5.7.44-31.65.4 (2024-09-)
+
+This release is part of {{post}}. The fixes are available to [MySQL 5.7 Post-EOL Support from Percona customers].
+
+We recommend upgrading to MySQL 8.0 or staying on 5.7; we'll support you.
+
+## Release Information
```
Comment (alina-derkach-oaza) [inline docs/release-notes/5.7.44-31.65.4.md]: Release Information -> Release information

File: docs/release-notes/5.7.44-31.65.4.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+* [Bug #35894664]: The result from a spatial index that included a column with a spatial reference identifier (SRID) was empty. Additionally, using FORCE INDEX to enforce a covering index scan on the spatial index caused an assertion error.
+
+* [Bug #33142654], [Bug #36543458]: Adds an additional check on the QT_NO_TABLE flag before accessing table_name to fix HEAP-USE-AFTER-FREE errors.
+
+* [Bug #33057164]: The PROCESSLIST_INFO column of THREADS did not update when a prepared statement was executed.
+
+* [Bug #34800754]: An InnoDB assertion error happened due to an invalid column index, even though the column index was valid.
+
+* [Bug #36252805]: To check if the binlog cache was empty, it had to be created first, which made it empty. This caused sessions that never needed a binlog cache to get one, using up resources unnecessarily.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/5.7.44-31.65.4.md]: the cache had to be created first

**Review by kamil-holubicki — APPROVED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by VarunNagaraju — APPROVED** [review]
LGTM

**Pull request merged**